sdk/storage/azure-storage-file-share/tests/perfstress_tests/T1_legacy_tests/upload.py

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import uuid
+
+from azure_devtools.perfstress_tests import RandomStream, get_random_bytes
+
+from ._test_base import _LegacyShareTest
+
+
+class LegacyUploadTest(_LegacyShareTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        self.file_name = "sharefiletest-" + str(uuid.uuid4())
+
+    def run_sync(self):
+        data = RandomStream(self.args.size)
```

Comment:
Can RandomStream be reused?

Reply:
Not currently - I would need to add some kind of "reset" feature to it - which would end up being as about as expensive as simply re-instantiating I think...